json-spec/extensions/stac-collection-spec.md

```diff
@@ -0,0 +1,154 @@
+# STAC Collection Extension Spec
+
+A group of STAC `Item` objects from a single source usually shared common metadata. This is especially true with satellite imagery that uses the STAC EO Extension. Rather than including these common metadata fields on every `Item`, they can be grouped into a `Collection`.
+
+## Collection Extension Description
+
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| collection_name | string | Collection Name | A name given to the Collection
+| collection_description | string (optional) | Collection Description | A human readable description of the collection
+
+A `Collection` does not have many specific fields, as it may contain any fields that are in the core spec as well as any other extension. If a field is specified in `Collection`, it will be ignored in any `Item` that links to that `Collection`. This is important because a `Collection` is the metadta that is common across all `Item` objects. If a field is variable at all, it should be part of the `Item` record.
+
+#### Linking to a `Collection`
+An `Item` links to a collection by providing a "link" with "rel" set to "collection". For example
+```
+{ "rel": "collection", "href": "link/to/collection/record" }
+```
+
+#### Using a `Collection`
+The fields from the `Collection` record can be merged with an `Item` record to get the complete metadata for the `Item`.
```

Comment:
Should make it clear if it's just properties that get merged, or if every field in the collection gets merged. Like it's not clear if the 'bands' below would be in the 'merged' full item. I think you intend it to be, but should make it clear in the doc.

Reply:
Ah yes good point. Yes the idea was that everything gets merged. 

I think it's a good approach overall, because it means we don't have to have confusing specs for what goes in a Collection record, it's up to the provider to decide + it's easy to make a complete Item record from both of them.

Reply:
Yeah, I like it. Just need to make it clear. If you can add it in to the PR great, if not I'll merge in tomorrow and add it in as I understand it.